js/app.js

```diff
@@ -4,7 +4,7 @@ var d3 = require('d3');
 // webpack import functions
 import {load_dv_data} from './modules/data_loading';
 import {create_circles, create_color_scale_function, add_color_legend} from './modules/circles';
-import {clone_states, clone_legend, add_circle_selector, find_closest_point, add_placeholder, clicked} from './modules/interactivity';
+import {clone_states, clone_legend, add_circle_selector, find_closest_point, add_placeholder, clicked, add_siteInfo_text, hideSiteInfo} from './modules/interactivity';
```

Comment:
I don't know if there is a good way to get around that because you have to specifically export from the file and then import to this one. We might be able to do something like this but I don't know if that really simplifies anything. 

```
import * as utils from './utils.js'; 
...
utils.multiply(2,3)
```

https://stackoverflow.com/a/38469257

Reply:
Yeah, maybe not. Thanks for looking into it; what you have seems reasonable.

Reply:
@mwernimont do you know if there are any better solutions to having this very long string of function names to import? Is there a way to have it just import anything from that file that is part of the export string? If so, how does it get into the namespace?